extensions/AbuseFilter/abusefilter.tables.sqlite.sql

```diff
@@ -28,20 +28,20 @@ CREATE TABLE /*$wgDBprefix*/abuse_filter_action (
 CREATE INDEX afa_consequence ON /*$wgDBprefix*/abuse_filter_action (afa_consequence);
 
 CREATE TABLE /*$wgDBprefix*/abuse_filter_log (
-	afl_id BIGINT unsigned NOT NULL AUTO_INCREMENT,
-	afl_filter varchar(64) binary NOT NULL,
+	afl_id BIGINT unsigned NOT NULL PRIMARY KEY AUTOINCREMENT,
+	afl_filter varchar(64) NOT NULL,
 	afl_user BIGINT unsigned NOT NULL,
-	afl_user_text varchar(255) binary NOT NULL,
+	afl_user_text varchar(255) NOT NULL,
 	afl_ip varchar(255) not null,
 	afl_action varbinary(255) not null,
 	afl_actions varbinary(255) not null,
 	afl_var_dump BLOB NOT NULL,
 	afl_timestamp binary(14) NOT NULL,
 	afl_namespace int NOT NULL,
-	afl_title varchar(255) binary NOT NULL,
-	afl_wiki varchar(64) binary NULL,
+	afl_title varchar(255) NOT NULL,
+	afl_wiki varchar(64),
 	afl_deleted tinyint(1) NOT NULL DEFAULT 0,
-	afl_patrolled_by int unsigned NULL
+	afl_patrolled_by int
```

Comment:
Should we add these schema changes to `WikiaUpdated` script and apply them site-wide?

Reply:
This SQLite schema is only used in tests